setup.py

```diff
@@ -112,8 +112,14 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['numpy', 'matplotlib', 'scipy', 'poppy', 'astropy'],
-      setup_requires=['numpy', 'astropy'],
+      install_requires=[
+          'numpy>=1.10.0',
+          'matplotlib>=1.5.3',
```

Comment:
We can require something older, but unless we're adding another dimension to the build matrix we're only testing on the latest. Not specifying a version at all leads to surprises.